al_servicec.js

```diff
@@ -167,10 +167,35 @@ class EndpointsC extends AlServiceC {
     }
 }
 
+/**
+ * @class
+ * HTTPS client for Alert Logic Collector_status service.
+ *
+ * @constructor
+ * @param {string} apiEndpoint - Alert Logic API hostname.
+ * @param {Object} aimsCreds - Alert Logic API credentials object, refer to AimsC.
+ * @param {*} retryOptions
+ */
+class CollectorStatusC extends AlServiceC {
```

Comment:
Suggestion: Move the Collector Status code to azcollectc.js as it would be consistent and we are sending status to azcollect

Reply:
I will create separate file collector_statusc.js .